### DISCUSSION TITLE
brute force detection with update password required action

### DISCUSSION THREAD

**Discussion by tonythomas1** [keycloak/keycloak#32482] [Q&A]
i want to have the update password via email required action to be done when user has 5 failed login attempts, currently i have the brute force applied but how can i implement the email and password reset

**Comment by darius-m** [comment]
Are you sure that would be a good idea? As long as the attacker does not gain access to the account, asking users to reset their password feels excessive. It also seems like an extremely easy way for an attacker to create a host of issues (e.g., email spam, user fatigue, among others), by simply performing a password spray attack.

**Reply by tonythomas1** [reply]
if we ignore the implications of what you said has this been implemented already by someone
